mobile/EmbeddedCounselDemo/EmbeddedCounselDemo.xcodeproj/project.pbxproj

```diff
@@ -10,9 +10,22 @@
 		EC4B2CD12DC4131700DBF1C1 /* EmbeddedCounselDemo.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = EmbeddedCounselDemo.app; sourceTree = BUILT_PRODUCTS_DIR; };
 /* End PBXFileReference section */
 
+/* Begin PBXFileSystemSynchronizedBuildFileExceptionSet section */
+		EC4B2B9F2DD3E81B00A629A3 /* Exceptions for "EmbeddedCounselDemo" folder in "EmbeddedCounselDemo" target */ = {
+			isa = PBXFileSystemSynchronizedBuildFileExceptionSet;
+			membershipExceptions = (
+				Info.plist,
+			);
+			target = EC4B2CD02DC4131700DBF1C1 /* EmbeddedCounselDemo */;
+		};
+/* End PBXFileSystemSynchronizedBuildFileExceptionSet section */
+
 /* Begin PBXFileSystemSynchronizedRootGroup section */
 		EC4B2CD32DC4131700DBF1C1 /* EmbeddedCounselDemo */ = {
 			isa = PBXFileSystemSynchronizedRootGroup;
+			exceptions = (
```

Comment:
what is this doing?

Reply:
this was just auto-generated when creating the customizable Info.plist